actors/market/src/lib.rs

```diff
@@ -1021,6 +1048,7 @@ fn compute_data_commitment<BS: Blockstore>(
 pub fn validate_and_return_deal_space<BS: Blockstore>(
     proposals: &DealArray<BS>,
     deal_ids: &[DealID],
+    next_id: DealID,
```

Comment:
This changes the exit code from VerifyDealsForActivation and ActivateDeals. Both of these are called by the miner actor. I think it does make sense to distinguish the exit code here, though, since the proposal ID was provided by a user and could indeed have expired. So 👍 .

Reply:
Change accepted, Done.